exec-example.ts

```diff
@@ -0,0 +1,11 @@
+import k8s = require('@kubernetes/typescript-node');
+import { Exec } from './exec';
+
+
+let command = process.argv[2];
+
+let kc = new k8s.KubeConfig();
+kc.loadFromFile(process.env['HOME'] + '/.kube/config');
+
+let exec = new Exec(kc);
+exec.exec('default', 'nginx-4217019353-tmc2j', 'nginx', command, process.stdout, process.stderr, process.stdin, true /* tty */);
```

Comment:
nit: can you remove the space at the end of the file and just keep the newline

Reply:
done (I think...)